src/main/java/org/example/homework6/BasicInteractiveCalculator.java

```diff
@@ -0,0 +1,44 @@
+package org.example.homework6;
+
+import java.util.Scanner;
+
+public class BasicInteractiveCalculator {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter a number");
```

Comment:
There is no logic related to 
5. The user is then given the option to continue entering numbers and operations for another calculation or to quit via entering (yes/no)
Please add loop with this functionality